src/templates/bespoke/presenter/presenter-view.tsx

```diff
@@ -73,9 +93,26 @@ const presenterView = (deck) => {
     return $cache[klass]
   }
 
+  const resizeNotes = (direction: string) => {
+    const current = parseFloat($(classes.noteContainer).style.fontSize) || 1
+    const intended = current + (direction === 'bigger' ? 0.1 : -0.1)
+    $(classes.noteContainer).style.fontSize = `${intended.toFixed(1)}em`
+  }
+
   const subscribe = (deck) => {
     // Next slide view
     $(classes.nextContainer).addEventListener('click', () => deck.next())
+    $(classes.noteButtonsBigger).addEventListener('click', () =>
+      resizeNotes('bigger')
+    )
+    $(classes.noteButtonsSmaller).addEventListener('click', () =>
+      resizeNotes('smaller')
+    )
+
+    document.addEventListener('keyup', (e: KeyboardEvent) => {
```

Comment:
Should use `keydown` event instead of `keyup`.

Check `e.repeat` within event callback to ignore key repeat.

Reply:
Something else is binding to keydown and stopping propagation.
What do you advise?

Reply:
The element for notes is stopping event propagation because global slide page navigation will conflict with arrow keys for scrolling notes. #431 is using `capture` flag: https://github.com/marp-team/marp-cli/blob/8b73b02de2988e85482133ea7387f771cff04d80/src/templates/bespoke/presenter/presenter-view.tsx#L166